### PR TITLE
#162727411 Setup redux

### PR DESCRIPTION
**Pull request by abejide001**
#### What does this PR do?
Setup redux
#### Description of Task to be completed?
- Setup the store
- Setup the actions
- Setup the reducer
- Setup the redux devtools
#### How should this be manually tested?
- Run `npm i`
- Run `npm start`
- Check Redux devtools for the intial state of the app
#### Any background context you want to provide?
Redux devtools extension for chrome needs to be installed
#### What are the relevant pivotal tracker stories?
[162727411](https://www.pivotaltracker.com/story/show/162727411)

### CODE REVIEW

**Review by wombolo — APPROVED** [review]
Looks Good

File: src/actions/authActions.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { TEST_DISPATCH } from './types';
+
+export const loginUser = userData => {
+    return {
+        type: TEST_DISPATCH,
+        payload: userData
+    }
+}
```
Comment (julietezekwe) [inline src/actions/authActions.js]: Please  insert end of line

File: src/actions/authActions.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { TEST_DISPATCH } from './types';
+
+export const loginUser = userData => {
```
Comment (julietezekwe) [inline src/actions/authActions.js]: I think it will be better if you can dispatch the action instead of returning it

Comment (abejide001) [reply src/actions/authActions.js]: okay, thanks.

File: src/actions/authActions.js (hunk 1 above)
Comment (abejide001) [reply src/actions/authActions.js]: okay

**Review by daniellamarr — APPROVED** [review]
Looks good

**Pull request merged**